src/app/left-bar/left-bar.component.html

```diff
@@ -57,7 +57,7 @@
 
       <left-bar-button
         [link]="'/' + globalVars.RouteNames.SEND_BITCLOUT"
-        [buttonLabel]="'Send BitClout'"
+        [buttonLabel]="'Send Clout'"
```

Comment:
```suggestion
        [buttonLabel]="'Send $CLOUT'"
```

Reply:
😍😍

Reply:
If you like these changes, accept them and this PR should be good to go.